pages/about-the-data.js

```diff
@@ -13,21 +14,89 @@ const About = () => (
+        Increasing transparency in the criminal justice system is of vital importance to us at the Texas Justice
+        Initiative. Our work relies on the use of information that can be released to the public.
+      </p>{' '}
+      <p>
+        Currently on our website, <strong>we compile data about three main incident types that occur in Texas</strong>:
       </p>
+      <ol>
+        <li>
+          <strong>Injury of officer(s) and/or civilian(s): </strong>Any shooting of or by a Texas peace officer,
+          occurring while the officer is on- or off-duty, causing an injury.
+        </li>
+        <li>
+          <strong>Death of officer(s) and/or civilian(s): </strong> Any fatal shooting of or by a Texas peace officer,
+          occurring while the officer is on- or off-duty.
+        </li>
+        <li>
+          <strong>Death in custody: </strong>
+          Any death of an individual while they are in a penal institution, in the custody of a peace officer or as a
+          result of a peace officer’s use of force, i n a jail, correctional facility, or state juvenile facility dies.
+        </li>
+      </ol>
       <p>
-        State laws allow anyone to get a good glimpse at the criminal justice system, though plenty of hurdles remain.
-        Through open records requests, TJI has obtained data from various governmental agencies. We then clean, analyze
-        and present the data – both in the form of a full data set and in comparative graphics – to the public. Here is
-        a description of our current data sets:
+        After an incident, the governmental agency in which it occurred (i.e. a county jail etc.) is required to file a
+        report for the respective incident (
+        <a
+          href="https://drive.google.com/file/d/1qRRN6HJpVTXvo4G5MzJGLwAHSPEHvNNR/view"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          <strong>1</strong>
+        </a>
+        ,{' '}
+        <a
+          href="https://drive.google.com/open?id=1iBJQ6zl1UQOEUnd18Krb9PnZEL8Nd0kY"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          <strong>2</strong>
+        </a>
+        ,{' '}
+        <a
+          href="https://drive.google.com/file/d/1MEEDDiIHfXdi6yM7bYoN8bROa8iYh8vP/view"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          <strong>3</strong>
+        </a>
+        ) with the Texas Office of the Attorney General (OAG) within 30 days.{' '}
+      </p>
+      <PipelineImage
+        src="/static/images/TJI_pipeline_Part1_horz_v4.svg"
+        alt="Diagram showing the process that begins once an incident occurs and ends when the Office of the Attorney General publishes a report"
+      />
+      <p>
+        The Texas OAG publishes the raw .PDFs of these reports. Through open records requests filed monthly, TJI obtains
+        data from the reports through the Texas OAG.
+      </p>
+      <p>
+        {' '}
```

Comment:
These are pretty silly right? ESLint seems to prefer using this method to add white space between text rather than a space in between html tags, i.e. `<p>{' '}Hi</p>` rather than `<p> Hi</p>`

Reply:
Weird! Why do we need it between the `<p>` and the text on the next line here but not in the paragraph below on line 79?